cmd/nri-flex/nri-flex-fips.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2019 New Relic Corporation. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build fips
+// +build fips
+
+package main
```

Comment:
same doubt i had here https://github.com/newrelic/infrastructure-agent/pull/1942#discussion_r1827743835

Reply:
> same doubt i had here [newrelic/infrastructure-agent#1942 (comment)](https://github.com/newrelic/infrastructure-agent/pull/1942#discussion_r1827743835)

Addressed. Thanks for pointing it out.

Reply:
have you tried by just adding an extra file only for fips? no main no nothing? I'm not sure if would work here but a minimal example on my environment works

Reply:
Yes, I was going to  comment the same, you can do as Guillermo says, leaving nri-flex as it is and just adding a new nri-flex-fips file with:

```

//go:build fips
// +build fips

package main

import (
	_ "crypto/tls/fipsonly"
)

```